jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/TargetSystemIntegrationTest.java

```diff
@@ -90,32 +85,20 @@ void afterEach() {
 
   @Test
   void endToEndTest() {
+    int jmxPort = PortSelector.getAvailableRandomPort();
 
     target =
-        createTargetContainer(JMX_PORT)
-            .withLogConsumer(new Slf4jLogConsumer(logger))
+        createTargetContainer(jmxPort)
+            .withLogConsumer(new Slf4jLogConsumer(targetSystemLogger))
```

Comment:
@robsunday here we don't need to find an available port on host as this will be used between containers anyway (and we don't connect anymore from docker host), see 
https://github.com/robsunday/opentelemetry-java-contrib/pull/2 to simplify this.